cmd/cli/plugin/package/display_utils.go

```diff
@@ -44,10 +47,17 @@ func DisplayProgress(initialMsg string, pp *tkgpackagedatamodel.PackageProgress)
 	}
 
 	s.Suffix = fmt.Sprintf(" %s", initialMsg)
-	s.Start()
 
+	fi, _ := os.Stdout.Stat()
+	// Start the spinner only if connected to terminal
```

Comment:
The comment describes what we should do but I think could we look at using a dependency like https://github.com/mattn/go-isatty or similar which is more comprehensive.

Reply:
Updated.